platform/src/test/kotlin/org/stellar/anchor/platform/config/Sep24ConfigTest.kt

```diff
@@ -23,8 +23,8 @@ class Sep24ConfigTest {
   @BeforeEach
   fun setUp() {
     secretConfig = mockk()
-    every { secretConfig.sep24MoreInfoUrlJwtSecret } returns "interactive url jwt secret"
-    every { secretConfig.sep24InteractiveUrlJwtSecret } returns "more_info url jwt secret"
+    every { secretConfig.sep24MoreInfoUrlJwtSecret } returns "more_info url jwt secret"
```

Comment:
Good catch! 👍 

This is a bug. Please submit a different PR for this bug fix.

Reply:
PR created and merged